tests/test_webclient.py

```diff
@@ -14,18 +15,21 @@
 
 from scrapy.core.downloader import webclient as client
 from scrapy.http import Request, Headers
+from scrapy.utils.python import to_bytes, to_unicode
 
 
-def getPage(url, contextFactory=None, *args, **kwargs):
+def getPage(url, contextFactory=None, r_transform=None, *args, **kwargs):
```

Comment:
a bit of nitpicking: perhaps `response_transform` would be a more appropriate name? :)